FWCore/ParameterSet/python/Config.py

```diff
@@ -554,6 +558,41 @@ def extend(self,other,items=()):
                 newSeq._place(name,self)
         self.__dict__['_Process__InExtendCall'] = False
 
+    # FIXME define explicitly if the items should be referenced or copied
```

Comment:
Here I'd love some feedback from a framework expert!
The goal is to load a Process object from a cff file into the main Process object.